stdlib/ballerina-http/src/main/java/org/ballerinalang/net/http/WebSocketUtil.java

```diff
@@ -172,21 +172,14 @@ public static void handleWebSocketCallback(Context context, CallableUnitCallback
         webSocketChannelFuture.addListener((ChannelFutureListener) future -> {
             Throwable cause = future.cause();
             if (!future.isSuccess() && cause != null) {
-                context.setReturnValues(createWebSocketConnectorError(context, future.cause().getMessage()));
+                context.setReturnValues(BLangVMErrors.createError(context, cause.getMessage()));
```

Comment:
Since we are using `BLangVMError` can we remove `createWebSocketConnectorError` method from `WebSocketUtil` ?

Reply:
Sorry my bad. You have already removed it. :)

Reply:
Already done